### PR TITLE
fix(manager): fixing myriad of issues that came with 2.5

### PR DESCRIPTION
**Pull request by ShlomiBalalis**

### CODE REVIEW

File: ccmlib/scylla_repository.py [hunk 1]
```diff
@@ -316,7 +316,10 @@ def setup_scylla_manager(scylla_manager_package=None):
                     mkdir -p dist/etc
                     mkdir schema
                     cp -r ./etc/scylla-manager/* ./dist/etc/
-                    cp -r ./etc/scylla-manager/cql ./schema/
+                    if [ -d "./etc/scylla-manager/cql" ]
```
Comment (juliayakovlev) [inline ccmlib/scylla_repository.py:319]: What may happen if schema won't be copied? Does it has affect?

**Review by juliayakovlev — COMMENTED** [review]
one question

File: ccmlib/scylla_repository.py (hunk 1 above)
Comment (ShlomiBalalis) [reply ccmlib/scylla_repository.py:319]: Not at all, according to michal they moved it into the binary. That's also why we don't config the migrate_dir parameter anymore

**Review by juliayakovlev — APPROVED** [review]

**Review by bentsi — APPROVED** [review]

**Comment by roydahan** [comment]
@ShlomiBalalis Do we need to backport it to manager-2.5 branch?
Do we even have branches for manager in ccm?

**Comment by ShlomiBalalis** [comment]
> @ShlomiBalalis Do we need to backport it to manager-2.5 branch?
> Do we even have branches for manager in ccm?

@roydahan We do not, we only use master

**Pull request merged**